lib/sheets/index.ts

```diff
@@ -50,6 +51,12 @@ class Sheets {
     results.forEach(data => {
       const getTiming = (key: string) => data.timings.find(t => t.id === key).timing;
       const dateObj = new Date(data.generatedTime);
+      let viewerUrl = '';
+
+      if(typeof data.fileName !== 'undefined' && typeof data.fileName !== 'undefined') {
```

Comment:
So you don't need to be concerned of that because `undefined` is `false` when type checked in a boolean scenario.